ansible/roles/airflow/templates/airflow.cfg.j2

```diff
@@ -0,0 +1,350 @@
+[core]
```

Comment:
I am not sure if these are all required or if we can be eliminate the ones we do not set, and they seemed well organized (they are currently organized thematically by default). I agree it's hard to see which ones we are controlling and which we are not though, so I am curious if you have alternative organizational options you would recommend.

Reply:
I think it depends how airflow handles this file. If you leave an option out will it use a default? If that's the case then I'd suggest removing anything that's not being set by us + any comments that aren't necessary